lib/internal/timers.js

```diff
@@ -0,0 +1,60 @@
+'use strict';
+
+// Timeout values > TIMEOUT_MAX are set to 1.
+const TIMEOUT_MAX = 2147483647; // 2^31-1
+
+module.exports = {
```

Comment:
I believe most of the other files have it at the bottom for a couple of reasons: it's easier to find and it works for situations where you are exporting something whose value is only set later on in the file.